packages/types/src/amount.ts

```diff
@@ -77,10 +77,28 @@ export const formatNumber = (number: number, options: FormatOptions): string =>
     : parseFloat(number.toFixed(precision)).toString();
 };
 
-export const formatAmount = (amount: Amount, exponent = 0) =>
-  fromBaseUnitAmount(amount, exponent)
-    .toFormat(6)
-    .replace(/(\.\d*?[1-9])0+$|\.0*$/, '$1');
+export const removeTrailingZeros = (strNum: string): string => {
+  return strNum.replace(/(\.\d*?[1-9])0+$|\.0*$/, '$1');
+};
+
+export const formatAmount = ({
+  amount,
+  exponent = 0,
+  commas = false,
```

Comment:
I change the comma default to be `false`. Up for discussion if reviewer feels it should be opposite. Potentially safer this way (?).